changelog/7077.breaking.rst

```diff
@@ -0,0 +1,19 @@
+Rewrote ``~sunpy.net.scraper.Scraper`` functions to use ``parse`` instead of regex and require only a single parse-formatted ``pattern`` variable while creating a Scraper object, instead of the two ``baseurl`` and ``pattern`` variables before.
```

Comment:
This changelog is more of a summary of work. 

We want our changelogs to be user facing, so we should detail what changed and how it breaks current workflows and how one would recover from that breaking changing.

Reply:
i've updated the changelog to reflect this, I wasn't sure whether to include the timerange case or not though (i did for now)